### PR TITLE
[Tables] Adding swagger.md 

### PR DESCRIPTION
**Pull request by seankane-msft**
Adding swagger.md file and adding directive to change TableEntityProperties to a `[][]byte`

### CODE REVIEW

**Comment by chamons** [comment]
I think this should be in Draft?

**Comment by seankane-msft** [comment]
> I think this should be in Draft?

@chamons ready for your review now :)

**Review by chamons — COMMENTED** [review]
How about naming it autorest.md or readme.md? Is there a reason for this specific name?

File: sdk/tables/swagger.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+<!-- Original autorest command used by Chris Scott -->
+<!-- autorest --use=@autorest/go@4.0.0-preview.20 https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cosmos-db/data-plane/readme.md --tag=package-2019-02 --file-prefix="zz_generated_" --modelerfour.lenient-model-deduplication --license-header=MICROSOFT_MIT_NO_VERSION --output-folder=aztables --module=aztables --openapi-type="data-plane" --credential-scope=none -->
+
+``` yaml
+go: true
+version: "^3.0.0"
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/d744b6bcb95ab4034832ded556dbbe58f4287c5b/specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json
+license-header: MICROSOFT_MIT_NO_VERSION
+clear-output-folder: false
```
Comment (chamons) [inline sdk/tables/swagger.md:13]: Half of this belongs in a common swagger that gets required, but we can extract that out later when there are two copies.

File: sdk/tables/swagger.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+``` yaml
+directive:
+  # dynamically change TableEntityProperties from map[string]interface{} to []byte
+  - from: swagger-document
+    where: $.definitions.TableEntityProperties
+    transform: >-
+      $["type"] = "string";
+      $["format"] = "byte";
+      delete $.additionalProperties;
+      $lib.log($);
```
Comment (chamons) [inline sdk/tables/swagger.md:32]: No need to debug print.

File: sdk/tables/swagger.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      return $;
```
Comment (chamons) [inline sdk/tables/swagger.md:33]: or return (I think?)

**Comment by seankane-msft** [comment]
> How about naming it autorest.md or readme.md? Is there a reason for this specific name?

Changed to autorest.md, no reason for this specific naming.

**Comment by seankane-msft** [comment]
@chamons 👍 ?

**Review by chamons — APPROVED** [review]
Sorry, thought I +1'ed it already.

**Pull request merged**